docs/ops/reduction/ReduceL2_4.md

```diff
@@ -0,0 +1,65 @@
+## ReduceL2 <a name="ReduceL2"></a>
+
+**Versioned name**: *ReduceL2-4*
+
+**Category**: *Reduction*
+
+**Short description**: *ReduceL2* operation performs reduction with finding the L2 norm of the 1st input tensor in slices specified by the 2nd input.
+
+**Attributes**
+
+* *keep_dims*
+
+  * **Description**: If set to `True` it holds axes that are used for reduction. For each such axis, output dimension is equal to 1.
+  * **Range of values**: True or False
+  * **Type**: `boolean`
+  * **Default value**: False
+  * **Required**: *no*
+
+**Inputs**
+
+* **1**: Input tensor x of any data type that has defined L2 norm operation. **Required.**
+
+* **2**: Scalar or 1D tensor with axis indices for the 1st input along which reduction is performed. **Required.**
```

Comment:
To be inline with ONNX, we should support negative values for `axis`

Reply:
@achetver , please, specify the range of possible values. It should be `[-r, r - 1]` where `r` is the rank of input tensor.

Reply:
Done.